### DISCUSSION TITLE
Change the default debugging baud rate

### DISCUSSION THREAD

**Discussion by HamzaHajeir** [espressif/arduino-esp32#8304] [Q&A]
Hi there
I'm monitoring my ESP32 with a rate of 921600, but in restarts it prints garbage.  
Such as:
`��[...]`

How can I change the baud of default debuging to the baud rate needed?

Thanks

**Comment by me-no-dev** [comment]
the ROM prints at baud 115200 and it's not something that can be changed unfortunately

**Reply by HamzaHajeir** [reply]
Isn't it something could be changed by a CONFIG ??  
As [CONFIG_ESP_CONSOLE_UART_BAUDRATE](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/kconfig.html#config-esp-console-uart-baudrate):
> This baud rate is used by both the ESP-IDF Bootloader and the app (including boot log output and default standard input/output/error of the app).

So I think it's something changeable, but not by the pre-built Arduino core, am I correct?

**Reply by me-no-dev** [reply]
we have turned off the logging from the second stage bootloader (and that is the one that can be controlled from config). What you see is coming from the ROM and can not be controlled. You could turn it off by pulling a bootstrap pin GPIO15 LOW: https://docs.espressif.com/projects/esptool/en/latest/esp32/advanced-topics/boot-mode-selection.html#other-pins
